app/views/planning_applications/index.html.erb

```diff
@@ -35,13 +35,17 @@
 <div class="status-bar-container">
   <h2 class="govuk-heading-m">Status update:</h2>
   <div class="status-bar">
-    <div class="status-panel" id="reviewer-requests">
-      <h3 class="govuk-heading-s"><%= t ".reviewer_requests.title" %></h3>
-      <p class="govuk-heading-m"><%= t ".reviewer_requests", count: @search.current_planning_applications.to_be_reviewed.count %></p>
+    <div class="status-panel<% if filter_enabled_uniquely?(status: 'to_be_reviewed') %> status-panel--highlighted<% end %>" id="reviewer-requests">
+      <a class="status-filter-link" href="<%= planning_applications_path(filter_enabled_uniquely?(status: "to_be_reviewed") ? {} : { status: [:to_be_reviewed] }) %>">
+        <h3 class="govuk-heading-s"><%= t ".reviewer_requests.title" %></h3>
+        <p class="govuk-heading-m"><%= t ".reviewer_requests", count: @search.current_planning_applications.to_be_reviewed.count %></p>
+      </a>
     </div>
-    <div class="status-panel" id="not-started">
-      <h3 class="govuk-heading-s"><%= t ".prior_approvals.title" %></h3>
-      <p class="govuk-heading-m"><%= t ".prior_approvals", count: @search.current_planning_applications.prior_approvals.not_started.count %></p>
+    <div class="status-panel<% if filter_enabled_uniquely?(application_type: 'prior_approval') %> status-panel--highlighted<% end %>" id="not-started">
+      <a class="status-filter-link" href="<%= planning_applications_path(filter_enabled_uniquely?(application_type: "prior_approval") ? {} : { application_type: [:prior_approval] }) %>">
+        <h3 class="govuk-heading-s"><%= t ".prior_approvals.title" %></h3>
+        <p class="govuk-heading-m"><%= t ".prior_approvals", count: @search.current_planning_applications.prior_approvals.not_started.count %></p>
+      </a>
     </div>
   </div>
 </div>
```

Comment:
As an improvement I think if we don't have any `planning_applications.to_be_reviewed` and `prior_approvals. not-started`

Reply:
Also I think it would make sense to move it in a component and we won't have the method in helpers

Reply:
> As an improvement I think if we don't have any `planning_applications.to_be_reviewed` and `prior_approvals. not-started`

I'm not sure what you mean here?

> Also I think it would make sense to move it in a component and we won't have the method in helpers

I thought it might be better to do this only when we need to dynamically create the filter links. Right now there will only ever be precisely two, statically defined.